contracts/ForkableRealityETH_ERC20.sol

```diff
@@ -2,977 +2,173 @@
     }
 
-    /// @notice Returns the questions's content hash, identifying the question content
-    /// @param question_id The ID of the question
-    function getContentHash(bytes32 question_id) public view returns (bytes32) {
-        return questions[question_id].content_hash;
-    }
-
-    /// @notice Returns the arbitrator address for the question
-    /// @param question_id The ID of the question
-    function getArbitrator(bytes32 question_id) public view returns (address) {
-        return questions[question_id].arbitrator;
-    }
-
-    /// @notice Returns the timestamp when the question can first be answered
-    /// @param question_id The ID of the question
-    function getOpeningTS(bytes32 question_id) public view returns (uint32) {
-        return questions[question_id].opening_ts;
-    }
-
-    /// @notice Returns the timeout in seconds used after each answer
-    /// @param question_id The ID of the question
-    function getTimeout(bytes32 question_id) public view returns (uint32) {
-        return questions[question_id].timeout;
-    }
-
-    /// @notice Returns the timestamp at which the question will be/was finalized
-    /// @param question_id The ID of the question
-    function getFinalizeTS(bytes32 question_id) public view returns (uint32) {
-        return questions[question_id].finalize_ts;
-    }
-
-    /// @notice Returns whether the question is pending arbitration
-    /// @param question_id The ID of the question
-    function isPendingArbitration(
-        bytes32 question_id
-    ) public view returns (bool) {
-        return questions[question_id].is_pending_arbitration;
-    }
-
-    /// @notice Returns the current total cumulative bonds
-    /// @dev Used when migrating questions in forks, as we have to give the child contract the money to pay out
-    /// @param question_id The ID of the question
-    function getCumulativeBonds(
-        bytes32 question_id
-    ) public view returns (uint256) {
-        return questions[question_id].cumulative_bonds;
-    }
-
-    /// @notice Returns the current best answer
-    /// @param question_id The ID of the question
-    function getBestAnswer(bytes32 question_id) public view returns (bytes32) {
-        return questions[question_id].best_answer;
-    }
-
-    /// @notice Returns the history hash of the question
-    /// @param question_id The ID of the question
-    /// @dev Updated on each answer, then rewound as each is claimed
-    function getHistoryHash(bytes32 question_id) public view returns (bytes32) {
-        return questions[question_id].history_hash;
-    }
-
-    /// @notice Returns the highest bond posted so far for a question
-    /// @param question_id The ID of the question
-    function getBond(bytes32 question_id) public view returns (uint256) {
-        return questions[question_id].bond;
+    // TODO: Make sure this gets called on initiateFork, it can't wait until executeFork because we can't arbitrate anything else that happens
```

Comment:
This would be a bigger rewrite, since during initiate fork the new tokens are not yet exsiting. 

why do we have to arbitrate anything else during that time?

Reply:
Yeah, probably easiest to do it like that. Freeze on initiate fork, then do the forking of the reality.eth after the fork.

Where this could be bad is if we propose an update then realize we got it wrong during the fork period. We can't replace it or make any other change to mitigate it. So try not to do that.

Reply:
>Where this could be bad is if we propose an update then realize we got it wrong during the fork period. We can't replace it or make any other change to mitigate it. So try not to do that.

I think this is fine. It's unlikely that we are realizing it during that 1 hour, if we have a long preparation time to think about an update in general.

Reply:
We cannot fork in 1 hour.